go.mod

```diff
@@ -28,7 +28,8 @@ require (
 	github.com/muesli/coral v1.0.0
 	github.com/muesli/mango-coral v1.0.1
 	github.com/muesli/roff v0.1.0
-	github.com/slack-go/slack v0.10.2
+	github.com/scylladb/go-set v1.0.2
+	github.com/slack-go/slack v0.10.1
```

Comment:
```suggestion
	github.com/slack-go/slack v0.10.2
```

Reply:
curious! I've adjusted the commit to keep v0.10.2 of slack 👍